cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -46,6 +46,7 @@ import com.here.ort.model.licenses.LicenseConfiguration
 import com.here.ort.model.licenses.orEmpty
 import com.here.ort.model.mapper
 import com.here.ort.model.readValue
+import com.here.ort.model.utils.SimplePackageConfigurationProvider
```

Comment:
Commit message needs some explanation of the new feature.

Reply:
Done.